spec/models/ticket_scanning_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe TicketScanning do
+  let(:conference) { create(:conference) }
```

Comment:
do you really need all this variables? shouldn't some of them be created by other factories? If not, we may want to modify the factories. I mean just creating what we need the factories can create the rest. :thinking:  At least we should remove the unneeded ones, if any. Improving the factories could be done in another PR.

Reply:
Yes I guess right now we don need all these variables. But yes some of them can be removed by modifying some factories. 👍

Reply:
:+1: